### PR TITLE
Change "Enable AntiDebug" label to "Disable Debugging Protections" to make it clearer

### PR DESCRIPTION
**Pull request by redstrate**
You need to enable this to allow debugging, but the label has the negative which doesn't make sense.

### CODE REVIEW

**Comment by KazWolfe** [comment]
Rejecting out of hand - while this is a correct (and valid) thing to change, it has a lot of history behind it that it has basically become a form of Easter egg or in-joke for the community.

**Comment by NotNite** [comment]
not sure I agree with this being closed, just because it's a very easy thing to change and "we have a running injoke of explaining a hard-to-understand label in our discord" is kind of weird

**Comment by redstrate** [comment]
I'm also confused, because I recently had to enable this option and was really confused as to whether or not it was enabled. It wasn't until later and some more reading that I realized "AntiDebug" is the name of the mechanism that strips away the debug protections and you're "enabling" that.

If this is intentionally like this, maybe we should put a comment explaining that it's not meant to be changed?

**Comment by KazWolfe** [comment]
Likely a good idea - I'll defer to Goat here since it was Word of Goat that I not change this in the past.

**Comment by goaaats** [comment]
It's probably time to say goodbye to AntiDebug, but I feel like if we are changing the text, we might as well change it to what it actually does - "Disable debugging protections" or some such.

**Comment by redstrate** [comment]
Changed to "Disable Debugging Protections" now, thx. I was thinking even "Disable anti-debugging protections" to still channel the name, but that's probably way too long for the menu

**Review by KazWolfe — APPROVED** [review]
T/N: Rename the service for apiX.

Still will require Goat sign-off.

**Pull request merged**
